.github/workflows/trigger_pr.yml

```diff
@@ -29,4 +48,7 @@ jobs:
               -H "Authorization: Bearer ${{ steps.generate-token.outputs.token }}" \
               -H "X-GitHub-Api-Version: 2022-11-28" \
               https://api.github.com/repos/$repo_owner/$repo_name/dispatches \
-              -d "{\"event_type\": \"$event_type\", \"client_payload\": {\"commit_hash\": \"$GITHUB_SHA\"}}"
+              -d "{\"event_type\": \"$event_type\", \"client_payload\": {\"commit_hash\": \"$GITHUB_SHA\" \
```

Comment:
Would it make sense to add the short-sha and use it in the name of the PRs in sds-shared-library? I think it will make the PR names a bit more pretty

Reply:
I will in an other PR, still need the long one for rust

Reply:
yep it will be only useful for the PR title